src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj

```diff
@@ -25,7 +25,7 @@
   <ItemGroup>
     <MauiImage Include="Resources\Images\*" />
     <MauiIcon Include="Resources\appicon.svg" ForegroundFile="Resources\appiconfg.svg" Color="#512BD4" />
-    <MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />
+    <MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" BaseSize="128,128" />
```

Comment:
Do we have any plans for best practices / docs on how to efficiently use images in Maui?